milhoja/__init__.py

```diff
@@ -1,7 +1,7 @@
-# -*- coding: utf-8 -*-
-
-__author__ = """Raphael Medaer"""
-__email__ = 'raphael@medaer.me'
 __version__ = '0.3.0'
```

Comment:
since we forked and renamed it, do we want to change the version now to something else? `1.0.0` for instance?

Reply:
I've handled this in a later PR :) Thanks for chasing me up though.